SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -40,6 +40,7 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
                 "OnlineSubsystemUtils",
                 "InputCore",
                 "Sockets",
+                "ReplicationGraph"
```

Comment:
Slightly worried that we now have a dependency on ReplicationGraph, even though it is not strictly required for us to function, but I guess the solution would be to create a SpatialReplicationGraph module which seems like a lot of unnecessary work.

Reply:
Yeah I'm not so worried, it's a core runtime module, and only has a handful of files. Also, most spatial games should be making use of replication graphs going forward.

Reply:
We may also make it a conditional dependency if needed?